internal/services/appservice/service_plan_resource.go

```diff
@@ -185,8 +185,8 @@ func (r ServicePlanResource) Create() sdk.ResourceFunc {
 			}
 
 			if servicePlan.MaximumElasticWorkerCount > 0 {
-				if !strings.HasPrefix(servicePlan.Sku, "EP") && !strings.HasPrefix(servicePlan.Sku, "PC") {
-					return fmt.Errorf("`maximum_elastic_worker_count` can only be specified with Elastic Premium Skus")
+				if strings.HasPrefix(servicePlan.Sku, "B") || strings.HasPrefix(servicePlan.Sku, "F") {
```

Comment:
Per https://github.com/hashicorp/terraform-provider-azurerm/blob/dce4255bb81d572ce8476d37938b304f2775d128/internal/services/appservice/helpers/service_plan.go#L54-L65, have you tested the other skus and confirmed that they all support `maximum_elastic_worker_count`?

Reply:
After testing, I think it's better to just add `WS` to allow list to scale out.